common/app/routes/Challenges/challenges.less

```diff
@@ -1,6 +1,8 @@
 // should be the same as the filename and ./ns.json
 @ns: challenges;
 
+@import (inline) '../../../../node_modules/codemirror/theme/dracula.css';
```

Comment:
We grab codemirror css from a cdn here: https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/server/views/partials/react-stylesheets.jade